src/main/java/com/opentable/server/ServerLoggingConfiguration.java

```diff
@@ -28,6 +30,8 @@
 
 /* Registered via META-INF/spring.factories to capture early application lifecycle events */
 class ServerLoggingConfiguration implements ApplicationListener<ApplicationEvent> {
+    public static final String DMITRY_COMPONENT_NAME_KEY = "info.component"; // Taken from Dmitry's stack
+    public static final String OT_COMPONENT_NAME_KEY = "ot.component.id"; // Taken from Dmitry's stack
```

Comment:
Where do these get set?

Reply:
in any spring compliant config file etc.